src/room.rs

```diff
@@ -1807,6 +1807,20 @@ impl PeerEventHandler for InnerRoom {
                 peer.scrape_and_send_peer_stats().await;
             }
         }
+
+        if let Some(peer_state) = self.peers.state().get(peer_id) {
+            peer_state
+                .get_tracks_ids()
+                .into_iter()
+                .filter_map(|track_id| {
+                    self.connections.get_by_track_id(&track_id)
+                })
+                .flatten()
+                .for_each(|conn| {
+                    conn.update_peer_state(peer_connection_state);
+                });
+        }
```

Comment:
This will overwrite peer connection state stored in Connection in SFU scenario with latest value, since Connection will be backed by two PeerConnections.

This is ok for now, but we will have to figure something out if we will be implementing quality score for SFU scenario.

Reply:
@alexlapa @turtletongue @evdokimovs I guess this is what causes [`room::room::patches_generation::disable_display_video` test to fail](https://github.com/instrumentisto/medea-jason/actions/runs/15947938305/job/44984561861?pr=212#step:9:630). Should be investigated.